operator/cmd/manager/mock.go

```diff
@@ -0,0 +1,43 @@
+package manager
+
+import (
+	"context"
+
+	"github.com/stretchr/testify/mock"
+	"sigs.k8s.io/controller-runtime/pkg/healthz"
+	"sigs.k8s.io/controller-runtime/pkg/manager"
+	"sigs.k8s.io/controller-runtime/pkg/webhook"
+)
+
+type MockManager struct {
```

Comment:
also here, I would suggest to replace this by a `moq` - generated implementation